integration-api/src/main/java/se/inera/intyg/minaintyg/integration/api/person/Person.java

```diff
@@ -12,6 +12,7 @@
 public class Person {
 
   private String personnummer;
+  private String namn;
```

Comment:
Could you remove fornamn, efternamn and mellannamn? If minaintyg only needs namn, then we can get rid of the rest.